
describe('practice-1-2', function () {



    it("todo:should throw exception when n < 0", function () {
        console.log("true function exec, answer is 292");
        var cout = count_change(100);
        console.log("your function exec, answer is "+cout);
        expect(cout).toBe(292);
    });
});




